tests/e2e_test.go

```diff
@@ -282,6 +305,165 @@ func TestSignedEntryTimestamp(t *testing.T) {
 	}
 }
 
+func setup(t *testing.T) {
+	if err := os.MkdirAll("testdata", 0755); err != nil {
+		t.Fatalf("Failed to create testdata directory: %v", err)
+	}
+
+	if _, err := os.Stat(testCAPath); os.IsNotExist(err) {
```

Comment:
you can remove this and the other `os.Stat` call - the temp dir is always cleaned up after a test finishes.